tests/cmdline/commands/test_computer.py

```diff
@@ -541,10 +541,40 @@ def test_computer_export_setup(self, tmp_path, sort):
         result = self.cli_runner(computer_export_setup, [sort, comp.label, already_existing_filename], raises=True)
         assert result.exit_code == ExitCode.CRITICAL
 
+    def test_computer_export_auto_filename(self, tmp_path):
+        """Test if 'verdi computer export setup' without providing an explicit filename"""
+
+        computer_label = 'test_computer'
+        self.comp_builder.label = computer_label
+        self.comp_builder.transport = 'core.ssh'
+        comp = self.comp_builder.new()
+        comp.store()
+
+        # Export setup without explicit filepath
+        exported_filename = tmp_path / f'{computer_label}-setup.yml'
+        cwd = os.getcwd()
+        os.chdir(tmp_path)
+        result = self.cli_runner(computer_export_setup, [comp.label])
+        assert result.exit_code == 0, 'Command should have run successfull.'
```

Comment:
This check is superfluous as it is done automatically by the `self.cli_runner` call